controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/util/Result.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2017 Pinterest, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.pinterest.rocksplicator.controller.util;
+
+/**
+ * A data structure for holding the result from TaskQueue so that
```

Comment:
because previously if we return null, the front end layer receives no content, not even null. But I just read a little bit about HTTP status code, we can use no_content_204 as the status code and therefore we don't need the Result class to bring the null to front end anymore. Thanks!